invenio_communities/templates/semantic-ui/invenio_communities/details/header.html

```diff
@@ -19,7 +19,17 @@
         </div>
 
         <div>
-          <h2 class="ui header">{{ community.metadata.title }}</h2>
+          <div class="flex align-items-center rel-mb-1">
+            <h2 class="ui header mb-0">{{ community.metadata.title }}</h2>
+            {% if community.access.visibility == 'restricted' %}
+              <div class="rel-ml-1 align-self-baseline">
```

Comment:
```suggestion
              <div class="rel-ml-1">
```
I think this is actually not needed if you remove the `align-items-center` from the `flex`-element? Also, I think the empty `<div>` around the `.flex`-element can be removed. Correct me if I'm wrong!